### PR TITLE
fix(ui): Eliminating signup page error

### PR DESCRIPTION
**Pull request by aniketkatkar97**
…API call.

### Describe your changes :

I worked on the signup page error issue.

#
### Type of change :

- [x] Bug fix

#
### Frontend Preview (Screenshots) :
<p align="center">
Screenshot 2022-08-01 at 1 22 46 PM
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/signup/index.tsx [hunk 1]
```diff
@@ -224,24 +200,6 @@ const Signup = () => {
...
-                            icon="info"
-                            title="Info"
-                            width="16px"
-                          />
-                        </div>
-                        <div className="tw-font-semibold tw-px-1">
-                          There is no team available.
-                        </div>
-                      </div>
-                    ) : null}
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/signup/index.tsx:244]: Please revert this changes as we need to show this banner

Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/pages/signup/index.tsx:244]: Move the badge to TeamsSelectable component.

**Review by chirag-madlani — APPROVED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]

**Comment by harshach** [comment]
@aniketkatkar97 are we showing all the teams listed here or only top 10?

**Comment by aniketkatkar97** [comment]
> 

We are showing all the teams we get from API

**Review by harshach — APPROVED** [review]

**Comment by harshach** [comment]
@chirag-madlani @aniketkatkar97 what is the limit param are we passing to search/suggest api?

**Comment by aniketkatkar97** [comment]
> @chirag-madlani @aniketkatkar97 what is the limit param are we passing to search/suggest api?

@harshach , We are passing size 10 for search/query API but no limit params for search/suggest, but we get maximum 5 items in response for search/suggest

**Comment by harshach** [comment]
if you pass 10 then you won't list all the teams available right? cc @chirag-madlani

**Comment by chirag-madlani** [comment]
@aniketkatkar97 please remove page_size from query api that we use to fetch teams

**Comment by chirag-madlani** [comment]
@aniketkatkar97 make sure that we are not limiting anything for frontend while query / search for teams.
Response of the teams depends on how many teams we have so it will be very.
